.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
-Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)
-
-- [ ] Tests and linter pass
+Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
+- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/{{metadata['repo']['name']}}/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
```

Comment:
The magic variables don’t seem to be working.

![D084A3CF-1D00-4CA0-85AC-B97154A3443E](https://user-images.githubusercontent.com/723048/65805493-c549e400-e153-11e9-8eb6-89899963e834.jpeg)